reports/summary-2020.md

```diff
@@ -0,0 +1,849 @@
+
+
+[SIG Multicluster]: https://git.k8s.io/community/sig-multicluster
+[SIG Usability]: https://git.k8s.io/community/sig-usability
+[Job Study]: https://docs.google.com/document/d/1lkPQdBEw-Xb5GEZ48WnpBgQdZ01EmTBhjcxJBlu5qJs/edit#heading=h.xsg4f6e6yk0p
+[SIG Contributor Experience]: https://git.k8s.io/community/sig-contributor-experience
+[SIG Architecture]: https://git.k8s.io/community/sig-architecture
+[production readiness survey]: https://docs.google.com/forms/d/e/1FAIpQLSc-J-Ydu5vp5G9vdvV5gBcraEDN_Bl-HSkVm15vAlU_orDvoA/viewform
+
+## Current Initiatives 
+
+This section summarizes current initiatives from each SIG and WG. Click on the 
+group for reported projects completed in 2020 and granular information for each 
+initiative with supporting links to KEPs and more. 
+
+- [SIG API-Machinery](https://git.k8s.io/community/sig-api-machinery/annual-report-2021.md#current-initiatives-and-project-health)
+    - Mitigating the impact of removing beta APIs in 1.22
+    - Server-side-apply to stable
+    - Server-side-apply client
+    - Optionally skip backend TLS verification
+    - Namespace labels
+    - CRD and admission webhook v1beta1 API removal: reminder on kubernetes-dev.
+    - Immutable fields API
+    - API unions
+    - Warnings to stable
+    - apiserver network proxy to beta
+    - Priority and fairness to stable
+
+- [SIG Apps](https://git.k8s.io/community/sig-apps/annual-report-2020.md#current-initiatives-and-project-health)
+    - Promoting CronJobs to GA
+    - Promoting PodDisruptionBudgets to GA
+
+- [SIG Architecture](https://git.k8s.io/community/sig-architecture/annual-report-2020.md#current-initiatives-and-project-health)
+    - Increased coverage of stable endpoints by conformance tests
+    - Coordinating dependency updates across projects
+    - Production Readiness Review process was made mandatory in 1.21, improving 
+    scalability, supportability, monitoring, and correct feature enablement
+    - Set up cross-project policies to move features towards stable ([conformance without beta](https://git.k8s.io/enhancements/keps/sig-architecture/1333-conformance-without-beta), [preventing "permabeta"](https://git.k8s.io/enhancements/keps/sig-architecture/1635-prevent-permabeta))
+    - Enhancements subproject is working with sig-release to assist SIGs in 
+    taking greater ownership of their KEPs during the release cycle
+
+- [SIG Auth](https://git.k8s.io/community/sig-auth/annual-report-2021.md#current-initiatives-and-project-health)
+    - BoundServiceAccountToken
+    - CSR v1
+    - Token Request / bound SA token admission
+    - client-go auth plugins
+    - external kubelet credential providers
+    - New features in Secrets Store CSI driver
+    - Pod Security Policy Replacement
+    - Several other KEPs going to General Availability on the report
+
+- [SIG Autoscaling](https://git.k8s.io/community/sig-autoscaling/annual-report-2020.md#current-initiatives-and-project-health)
+    - No report
```

Comment:
We've now got the SIG Autoscaling report merged (though have a [follow-up PR](https://github.com/kubernetes/community/pull/5819) to make the name match as per other reports):

```suggestion
    - Promoting HPA v2 to stable
    - Promoting HPAScaleToZero to beta
    - Vertical pod autoscaler adding support for customized recommenders
    - Cluster autoscaler adding support for gRPC custom cloud providers
```

Reply:
Thanks @gjtempleton !